Lib/abc.py

```diff
@@ -195,6 +229,8 @@ def __instancecheck__(cls, instance):
 
     def __subclasscheck__(cls, subclass):
         """Override for issubclass(subclass, cls)."""
+        if _C_speedup:
+            return _abc_subclasscheck(cls, subclass)
```

Comment:
To make it even faster you might want to try this:

```C
if _C_speedup:
    ABCMeta. __subclasscheck__ = _abc_subclasscheck
```

Reply:
Somehow this doesn't work, it complains that `_abc_subclasscheck`.expected two arguments but got one. I tried various flags etc., but still can't make it work. Do you have and idea how to fix this?

Reply:
I mean if I just do
```
def __subclasscheck__(cls, subclass):
    return _abc(cls, subclass)
```
then it works, but I can't just write `__subclasscheck__ = _abc_subclasscheck` in class body.

Reply:
Let's figure this out later.  It's a micro-optimization/cuter code anyways.

Reply:
And if you do two separate `PyABCMeta` and `CABCMeta` classes you won't have "if" anyways.

Reply:
OK, I did the split, will add the tests now.